app/src/main/java/org/mozilla/fenix/components/TopSiteStorage.kt

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.components
+
+import android.content.Context
+import androidx.lifecycle.LiveData
+import mozilla.components.feature.top.sites.TopSite
+import mozilla.components.feature.top.sites.TopSiteStorage
+import org.mozilla.fenix.test.Mockable
+
+@Mockable
+class TopSiteStorage(private val context: Context) {
+    var cachedTopSites = listOf<TopSite>()
+
+    private val topSiteStorage by lazy {
+        TopSiteStorage(context)
```

Comment:
Does this construct the fenix or AC `TopSiteStorage`? Seems like it builds the fenix one.

Reply:
This is building the AC TopSiteStorage.